src/proto/message_translation/power_frame_msg.hpp

```diff
@@ -112,6 +112,19 @@ TbotsProto_PowerStatus inline createNanoPbPowerStatus(
     return status;
 }
 
+/**
+ * Calculates width of pulse for chicker
+ * @param kick_constant
+ * @param kick_coefficient
```

Comment:
These params should have descriptions. e.g. "the constant to use in the kick speed to pulse width conversion"

Reply:
Also, if you take a look at other functions where `calculateChickerPulseWidth` is called, you'll see that `kick_constant`, `kick_coefficient`, `chip_constant`, etc. are undocumented in the function docs. This is outside the scope of this PR, but it might be worth updating those function docs with descriptions for those params